features/steps/advance_search.py

```diff
@@ -19,3 +24,65 @@ def step_impl(context):
     advanced_search_page = AdvancedSearchPage(context.browser)
     advanced_search_page.first_flaw_exist()
     context.browser.quit()
+
+
+@when('I select the {field} and {value} to search flaws')
+def step_impl(context, field, value):
+    advanced_search_page = AdvancedSearchPage(context.browser)
+    advanced_search_page.select_field_and_value_to_search(field, value)
+    advanced_search_page.click_search_btn()
+
+@then('I am able to view flaws matching the search with {field} as {value}')
+def step_impl(context, field, value):
+    advanced_search_page = AdvancedSearchPage(context.browser)
+    time.sleep(3)
```

Comment:
Check flaw list exist instead of use time.sleep, sleep is not reliable since the search time in uncertain.

Reply:
For example check flaw list element visibility, use visibility_of_element_located().

Reply:
Updated.